dsperse/src/compile/compiler.py

```
@@ -396,116 +375,101 @@ def _compile_slices(self, dir_path: str, input_file_path: Optional[str] = None,
+                # 4. Update model-level (in-memory) and slice-level (on-disk) metadata
+                if 'compilation' not in slice_data:
+                    slice_data['compilation'] = {}
+                slice_data['compilation'][be] = comp_block
+
+                slice_meta_path = Path(slice_dir) / "metadata.json"
+                if slice_meta_path.exists():
+                    CompilerUtils.update_slice_metadata(idx, slice_meta_path, success, comp_block, backend_name=be)
+
+                if success:
+                    successful_backends.append(be)
+            except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)